app/views/layouts/_navbar.html.erb

```diff
@@ -65,12 +65,12 @@
 
       <li role="presentation"><%= link_to "Manage Users", users_path, title: "Manage every Autolab user" %></li>
 
-      <li role="presentation"><%= link_to "Github Integration", github_integration_admin_path, title: "Manage Github Integration" %></li>
-
       <li role="presentation"><%= link_to "Manage API Applications", oauth_applications_path, title: "Manage registered API clients" %></li>
 
       <li role="presentation"><%= link_to "Bulk Email Instructors", email_instructors_admin_path, title: "Send a bulk email to every Autolab instructor" %></li>
 
+      <li role="presentation"><%= link_to "Configure Autolab", autolab_config_admin_path, title: "Configure Autolab settings" %></li>
+
```

Comment:
It's likely not related to your PR - but why is there dropdown2 and dropdown3?

Reply:
I think it's related to supporting mobile browsers, not 100% sure though.